test/yaml.ts

```
@@ -0,0 +1,80 @@
+}
+
+function prepareParseOption(options: YamlOptionsMultiple | undefined): any {
+	return {
+		prettyErrors: true,
+		uniqueKeys: !options?.removeTemplates,
+		strict: false,
+		...options,
+	};
+}
+
+class AggregateError extends Error {
```

Biome (1.9.4)

[error] Do not shadow the global "AggregateError" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)